src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,155 @@
+package duke.ui;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Encapsulates the user interface for Duke.
+ */
+public class Ui {
+    private Scanner scanner;
+    
+    static String ADD_TASK_LINE = ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>";
+    static String DONE_TASK_LINE = "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~";
+    static String LIST_TASK_LINE = "________________________________________________________";
+    static String BYE_LINE = "========================================================";
```

Comment:
Would it be better to name these as "LINE_XXX_XXX" instead?
Should these variables be final?
They appear to be constants and prefixing them with the same word will make them appear together when sorted alphabetically (see coding standard document)